include/maliput_malidrive/constants.h

```diff
@@ -44,6 +44,8 @@ static constexpr double kScaleLength{1.};         // [m]
 static constexpr double kBaseLinearTolerance{1e-6};  // [m]
 /// Multiplier used to increase the tolerance by the Builder.
 static constexpr double kToleranceStepMultiplier{1.1};
+/// Minimum length a lane should have so it can be shortened to add a new lane width.
+static constexpr double kMinLaneLengthToAdaptWidthDiscontinuity{1e-1};  // [m]
```

Comment:
nit: This is the default lenght of the adapting lane width function. When we find a lane's length that collides with that we do something, but that's just a consequence.

Change it to `kDefaultAdaptingFunctionLength`
and match the docstring.

Reply:
Done.